src/Components/WebAssembly/Server/src/DebugProxyLauncher.cs

```diff
@@ -110,6 +117,24 @@ private static void PassThroughConsoleOutput(Process process)
         {
             process.OutputDataReceived += (sender, eventArgs) =>
             {
+                // It's confusing if the debug proxy emits its own startup status messages, because the developer
+                // may think the ports/environment/paths refer to their actual application. So we want to suppress
+                // them, but we can't stop the debug proxy app from emitting the messages entirely (e.g., via
+                // SuppressStatusMessages) because we need the "Now listening on" one to detect the chosen port.
```

Comment:
Is this because it runs with an unknown port?

Reply:
I think the child process picks one at runtime based on availability.